astroquery/esa/hubble/core.py

```diff
@@ -63,7 +64,7 @@ def __init__(self, *, tap_handler=None, show_messages=True):
             self.get_status_messages()
 
     def download_product(self, observation_id, *, calibration_level=None,
-                         filename=None, verbose=False, product_type=None):
+                         filename=None, folder=os.getcwd(), verbose=False, product_type=None):
```

Comment:
I have replaced os.getcwd with None. For the output_folders, I created a new method so, in case folder is None, only the filename is returned, so it is downloaded in the current directory. If it is provided, then using os.path we get the absolute path.

Reply:
I'm afraid the change hasn't been committed for this method